modules/api/src/main/java/org/apache/fluo/api/data/Bytes.java

```diff
@@ -118,7 +155,7 @@ public Bytes() {}
    * @param b2 second byte sequence to compare
    * @return comparison result
    */
-  public static final int compareBytes(Bytes b1, Bytes b2) {
+  private static final int compareBytes(Bytes b1, Bytes b2) {
```

Comment:
Given that this is now private, how about moving it's description of how the compare works to the javadoc of the public method `compareTo(Bytes other)`?